deploy/helm/flightctl/templates/flightctl-prometheus-clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We should not install this on OCP, i.e.

{{ if .Values.prometheus.enabled }}
...
...